app/commands/sre.py

```diff
@@ -47,8 +49,12 @@ def sre_command(ack, command, logger, respond, client, body):
             webhook_helper.handle_webhook_command(args, client, body, respond)
         case "version":
             respond(f"SRE Bot version: {os.environ.get('GIT_SHA', 'unknown')}")
-        case "google-service":
-            google_service.google_service_command(client, body)
+        case "google":
+            if PREFIX == "dev-":
+                google_service.google_service_command(client, body, respond)
+            else:
+                respond("This command is only available in the dev environment.")
```

Comment:
I like that you added this here!

Reply:
Yeah, I figured I should setup a toggle instead of just leaving it hanging in there in prod 😅